### PR TITLE
add isFormFilledIn task to arrays section

### PR DESCRIPTION
**Pull request by ValeriiBoiko**

### CODE REVIEW

**Comment by ValeriiBoiko** [comment]
@dosandk could you review changes, please?

File: tracks/arrays/03-isFormFilledIn/readme.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# isFormFilledIn
+
+В даному завданні, потрібно реалізувати функцію, яка буде визначати чи заповнені обовʼязкові поля форми.
```
Comment (dosandk) [inline tracks/arrays/03-isFormFilledIn/readme.md:3]: 👍 good супер!

File: tracks/arrays/03-isFormFilledIn/readme.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# isFormFilledIn
+
+В даному завданні, потрібно реалізувати функцію, яка буде визначати чи заповнені обовʼязкові поля форми. 
+
+Функція прийматиме два параметри:
```
Comment (dosandk) [inline tracks/arrays/03-isFormFilledIn/readme.md:5]: 👍 good  супер!

File: tracks/arrays/03-isFormFilledIn/readme.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- перший, це обʼєкт із полями форми `{"fieldName1": "fieldValue1"}`. Для простоти всі значення полів мають тип string.
+- другим аргументом є масив полів, які обовʼязкові для заповнення (`["email", "name"]`)
+
+Якщо всі обовʼязкові поля мають значення, то функція має повернути true. Якщо якесь з обовʼязкових полів НЕ заповнене, то функція має повернути false. Стан необовʼязкових полів можна ігнорувати.
```
Comment (dosandk) [inline tracks/arrays/03-isFormFilledIn/readme.md:9]: 👍 good

File: tracks/arrays/03-isFormFilledIn/readme.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+```
+
+<details>
+  <summary>Підказка</summary>
+
+---
+
+  1) Оскільки необовʼязкові поля можна не брати до уваги, то можна використати масив обовʼязкових полів для ітерації. Якби потрібно було проводити додаткову валідацію на інших полях, то прийшлось би трансформувати обʼєкт форми в масив.
+
+  2) Для зручнішого визначення чи відповідає поле умові, і якщо ні, то перервати процес ітерації, можна використати метод масиву [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)
```
Comment (dosandk) [inline tracks/arrays/03-isFormFilledIn/readme.md:30]: 👍 good

**Review by dosandk — APPROVED** [review]
👍 good 
супер! хороша задача!

**Pull request merged**